docs/UserGuide.md

```diff
@@ -181,6 +181,16 @@ For example, given the command `add t/e d/Lunch a/15.60 da/15-08-2022 c/Food`, l
 - Letters followed by a forward slash is called a prefix and will be used to specify a specific parameter<br>
   e.g `t/ ` refers to the entry type parameter.
 
+- PennyWise will always take the **LAST OCCURRENCE** of a command prefix should there be more than 1 of the same command
+  prefix in a given command. 
+  e.g In `add t/i d/Monthly Allowance a/200.00 da/01-10-2022 c/Allowance d/Oct Allowance d/October Allowance`, PennyWise
```

Comment:
Minor typo of e.g when it should be e.g. seems to be an inconsistency throughout...

Reply:
Have fixed all the e.g.